docs/changelog.ptxt

```diff
@@ -9,9 +9,10 @@ OpenMSX 0.3.1
 -------------
 Change: Add copyright information to build system
 
-Fix #1210: Typo in readme.
```

Comment:
Since we have the GitHub identifiers now, I see no reason not to reference them, though I don't think we did in OpenSFX. (We do in OpenTTD itself.)